tests/api/enqueue.cpp

```diff
@@ -439,3 +439,147 @@ TEST_F(WithCommandQueue, FinishAfterFlush) {
         thread->join();
     }
 }
+
+#ifdef CLVK_UNIT_TESTING_ENABLED
+TEST_F(WithCommandQueue, EnqueueTooManyCommands) {
+
+    static const unsigned NUM_INSTANCES =
+        CLVK_CONFIG_GET(max_entry_points_instances);
```

Comment:
I was initially thinking we'd override this config option to reduce the max number of instances: we don't need that many for this test. It doesn't take a lot of time though so fine as is.